pkg/api/dashboard.go

```diff
@@ -277,10 +277,6 @@ func PostDashboard(c *m.ReqContext, cmd m.SaveDashboardCommand) Response {
 		return Error(500, "Failed to save dashboard", err)
 	}
 
-	if err == m.ErrDashboardFailedToUpdateAlertData {
```

Comment:
Looks good. But it doesn't solve the issue. The response in the issue says "Failed to save dashboard" which means the error was returned on line 277. Otherwise, the error message should have said "Invalid alert data. Cannot save dashboard" Or am I mistaken?